_docs_explore/51-backbone.md

```diff
@@ -10,6 +10,23 @@ The enmeshed Backbone embraces all central services required by the enmeshed pla
 
 One enmeshed Backbone is currently hosted and maintained by j&amp;s-soft GmbH.
 
+## Backbone Components (Runtime)
+
+The Backbone consists of two main components:
+
+- Admin API
+- Consumer API
```

Comment:
Event Handler is missing

Reply:
Doesn't it? 
Maybe we should define who the reader of this document is. 😉 For me it is the person hosting the Backbone. Because who else would be interested which infrastructure exists. 
And if my assumption is correct, then this person would also be interested in the existence of the Admin UI - at least in my opinion.

Reply:
Ok, makes sense.